packages/serverpod/lib/src/server/log_manager/log_manager.dart

```diff
@@ -127,7 +127,8 @@ class SessionLogManager {
       order: _nextLogOrderId,
     );
 
-    if (_session.serverpod.runMode == ServerpodRunMode.development) {
+    if (_session.serverpod.runMode == ServerpodRunMode.development ||
+        _session.serverpod.runMode == ServerpodRunMode.test) {
```

Comment:
I actually realized now that this (and on other place) doesn't respect the "enable logging" setting because it writes directly to `stdout`. I will remove this I think and we can revisit this.